apps/services/handlers.py

```diff
@@ -206,6 +207,21 @@ async def create_user_request_review(
                 "bed_place_id": user_request_review.bed_place_id,
             },
         )
+        if user_request_review.status_id == 1:
```

Comment:
in scope of eu-132 i didn't ask u to do it. so, pls delete new lines of code

Reply:
added for checking functionality